### ISSUE TITLE
Passwords for network drives are no longer stored.

### ISSUE THREAD

**Issue by gigabit942007** [W4RH4WK/Debloat-Windows-10#128]
When having a network drive if i want to reconnect I have to retype the password for every windows login, the password is saved in the Credential Manager but it isn't used, Windows says the the password is incorrect, but i'm confident the password is right, tried to change the password to things like "123" for testing, same result.

**Comment by W4RH4WK** [comment]
Have you encountered this problem on any other PC? I am using network drives everyday on 2 PCs (both setup with these scripts) and never encountered such an issue.

Is your system up-to-date? Which scripts did you run? Have you made any modifications to them prior?

**Comment by gigabit942007** [comment]
I have used most of the scripts, but the one that caught my eye was fix-privacy-settings.ps1, here i see the Credentials part for the groups part, it appears that this is the culprit, i think.

$groups = @(
    "Accessibility"
    "AppSync"
    "BrowserSettings"
    "Credentials" # could this be?
    "DesktopTheme"
    "Language"
    "PackageState"
    "Personalization"
    "StartLayout"
    "Windows"
)

The ones used are:
block-telemetry
disable-services
fix-privacy-settings
optimize-user-interface
optimize-windows-update
remove-default-apps
remove-onedrive

Fresh Windows install as up to date you can get. 1703 Latest patches, Windows 10 Pro N with the media pack installed.
I don't have an ideea how i can revert that for the moment, but if you give me a script i'm willing to test for you, or i'll research myself how to, but it's gonna take some time :)

**Comment by W4RH4WK** [comment]
If this setting really is the culprit, try re-enabling it. To do this, open the Registry Editor (Windows + r -> "regedit") and navigate to `Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\SettingSync\Groups\Credentials`. There should be an entry `Enabled` with its value set to `0`. Try changing this to `1` or deleting the `Enabled` entry completely. Reboot your computer and test whether things are working.

**Comment by gigabit942007** [comment]
I got it working, apparently somehow it got the wrong credentials in some cache, the steps i went trough were.
Installing Windows, mapping drive, running scripts, here after this, i changed the user for the drive, after this step it did not work anymore, not sure if it's the script because enabling or disabling this did not have any effect.
How i fixed it: removed all the credentials in the Credential Manager and then ran net use * /DELETE
This might have fixed it. Tried even reducing the cached logins, might actually be a windows bug because event trying different authentication protocols i could not get this working.
